### PR TITLE
_nsc-event-nextjs_sprint6_#285_nav_bar_signout_button

### PR DESCRIPTION
**Pull request by miqo1385**
Please fill out this description with the following:
Resolves <issue #285 >

https://github.com/SeattleColleges/nsc-events-nextjs/assets/97195623/4de46263-9336-411e-9f45-04203579bf2b

I fix the sign out button, however when click in the profile I'm not sure is redirecting correctly to either Admin or Creator. I would be super appreciated if somebody can give me a hand with that.

### CODE REVIEW

**Review by taylorpapke — CHANGES_REQUESTED** [review]
It looks like the functionality is working properly for the dropdown. It routes to admin/creator pages and signs outs as expected. However, we've lost event loading on this branch and there are some UI design changes to the nav bar that I believe to be outside the scope of this issue.

File: components/Navbar.tsx [hunk 1]
```diff
@@ -1,66 +1,180 @@
...
+                keepMounted
+                transformOrigin={{
+                  vertical: 'top',
+                  horizontal: 'right',
+                }}
+                open={Boolean(anchorElUser)}
+                onClose={handleCloseUserMenu}
+              >
+                <MenuItem onClick={handleCloseUserMenu}>
+                  <Link href={profileLink()} passHref>
```
[the remark this answers is not in the document]
Comment (miqo1385) [reply components/Navbar.tsx]: it was fixed.

**Comment by miqo1385** [comment]
NSC white logo added. I change Profile for My Account

https://github.com/SeattleColleges/nsc-events-nextjs/assets/97195623/2cdd55eb-68d2-4eaf-8f68-10dc40541775

**Comment by heosman** [comment]
@miqo1385 I looked over your changes on this PR again and it looks like the profile dropdown is working properly. I am routed to the appropriate pages when signed in as a admin or creator. Something you should know is that the white NSC logo has already been added to the nav bar on the main branch.

Like @taylorpapke mentioned, the events are not loading when using the NSC API URL. 
image

The "Event" link on the nav bar is causing an error because it is routing to the Event Detail page instead of the Home page. The Sign In link should still be visible on the bar when not signed in. The "My Account" link should not be appearing in the profile dropdown for normal users. The profile dropdown icon also needs to be updated, and the "NSC" link next to the NSC Logo should be removed, as I have mentioned before.

**Comment by taylorpapke** [comment]
@miqo1385 I think the problem here is that you are using a very old branch. I believe the best course of action would be to create a new branch from main and copy over the changes you want from this PR.

**Comment by MaddasaHatter** [comment]
Still seeing some conflicts and failed checks. I am also having the same issues as stated above so I am assuming you haven't had the chance to make the requested changes yet.

**Comment by MaddasaHatter** [comment]
Still seeing some conflicts and failed checks. I am also having the same issues as stated above so I am assuming you haven't had the chance to make the requested changes yet.
Screenshot 2024-06-10 225411

**Review by Robel-003 — COMMENTED** [review]
I'm able to sign out but when I click "My Account" link in the dropdown, I'm routed to the dashboard.

https://github.com/SeattleColleges/nsc-events-nextjs/assets/116779961/4e1d4efd-33f5-483d-9070-e92459181c63

**Pull request closed without merging**